### PR TITLE
obs-backgroundremoval: reinit at 0.5.16

### LINKED ISSUE #219332: Package request: obs-backgroundremoval
**Project description**

An OBS plugin for removing background in portrait images (video), making it easy to replace the background when screen recording. 

This package was removed in https://github.com/NixOS/nixpkgs/pull/207845
The build might be fixed as of https://github.com/royshil/obs-backgroundremoval/pull/150 
@nagy if you don't mind giving it a try

**Metadata**

* homepage URL: https://github.com/royshil/obs-backgroundremoval
* source URL: https://github.com/royshil/obs-backgroundremoval
* license: mit
* platforms: linux, darwin

### PR DESCRIPTION
**Pull request by Zahrun**
###### Description of changes

Closes https://github.com/NixOS/nixpkgs/issues/219332

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/video/obs-studio/plugins/obs-backgroundremoval/default.nix [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  version = "0.5.16";
+
+  src = fetchFromGitHub {
+    owner = "royshil";
+    repo = "obs-backgroundremoval";
+    rev = "v${version}";
+    hash = "sha256-E+pm/Ma6dZTYlX3DpB49ynTETsRS2TBqgHSCijl/Txc=";
+  };
+
+  nativeBuildInputs = [ cmake git ];
```
Comment (raphaelr) [inline pkgs/applications/video/obs-studio/plugins/obs-backgroundremoval/default.nix:22]:
```suggestion
  nativeBuildInputs = [ cmake ];
```
git isn't required to build this plugin (and the fetchFromGitHub result doesn't contain any git metadata anyway)

File: pkgs/applications/video/obs-studio/plugins/obs-backgroundremoval/default.nix [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, obs-studio
+, onnxruntime
+, opencv
+, git
```
Comment (raphaelr) [inline pkgs/applications/video/obs-studio/plugins/obs-backgroundremoval/default.nix:8]:
```suggestion
```

**Review by raphaelr — CHANGES_REQUESTED** [review]
Works just fine
<details>

demo

</details>

**Review by raphaelr — CHANGES_REQUESTED** [review]
Please [squash](https://git-rebase.io/) the "no need for git" commit into the "obs-backgroundremoval: reinit at 0.5.16" one

**Review by raphaelr — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
@ofborg build obs-studio-plugins.obs-backgroundremoval

**Pull request merged**